docroot/modules/humsci/hs_field_helpers/src/Plugin/Field/FieldType/DisplayModeField.php

```diff
@@ -112,7 +112,7 @@ public static function schema(FieldStorageDefinitionInterface $field_definition)
    *   Complete form.
    */
   public function elementValidate(array $element, FormStateInterface $form_state, array $form) {
-    $modes = &$form_state->getValue(['settings', 'allowed_values']);
+    $modes = &$form_state->getValue($element['#parents']);
```

Comment:
Curious what this change is doing and if it causes any validation issues for other displays?

Reply:
@joegl I was getting validation errors that prevented me from correctly setting the labels for each display mode every time I tried to create or edit any _Display Mode Select_ field. I did some debugging and found that the path for getting the display mode values was incorrect. My guess is that something changed with a Drupal update and we never realized because this validation only runs when adding or updating a _Display Mode Select_ field, which doesn't happen often. 

I checked other existing fields and validation is working as expected after the fix.

Reply:
Awesome thanks @cienvaras . Do you think it's worth writing a test for this?